src/types/bookshelf.ts

```diff
@@ -1,7 +1,17 @@
 import { APISummaryBook } from './book';
 
-export interface APISummaryBookshelf {
+export interface APISummaryBookshelf extends APIBookshelf {
   bookshelfId: number;
   bookshelfName: string;
+}
+
+export interface APIBookshelfBookList extends APIBookshelf {
+  count: number;
+  empty: boolean;
+  first: boolean;
+  last: boolean;
+}
```

Comment:
p1;

책장 책 조회할 때는 APIBook 타입에 담긴 정보들이 포함되니 이렇게 바꾸는게 좋을 것 같아요!

```suggestion
export interface APIBookshelfBookList {
  count: number;
  books: APIBook[];
  empty: boolean;
  first: boolean;
  last: boolean;
}
```